Author: Fabian Vogt <fabian@ritter-vogt.de>
Subject: Fix i18n call

The call to ki18n_install was autogenerated by a script, but
it requires I18n to be available.

---
 CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: kajongg-17.03.90/CMakeLists.txt
===================================================================
--- kajongg-17.03.90.orig/CMakeLists.txt
+++ kajongg-17.03.90/CMakeLists.txt
@@ -15,7 +15,7 @@ set(CMAKE_MODULE_PATH ${CMAKE_CURRENT_SO
 find_package(PythonInterp ${PYTHON_MIN_VERSION} REQUIRED)
 find_package(Twisted ${TWISTED_MIN_VERSION} REQUIRED)
 find_package(Qt5 ${QT_MIN_VERSION} REQUIRED NO_MODULE COMPONENTS Core Gui Svg Widgets)
-find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS Config DocTools)
+find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS Config DocTools I18n)
 
 include(FeatureSummary)
 include(ECMPackageConfigHelpers)
